Gopkg.lock

```diff
@@ -166,12 +166,6 @@
   packages = ["pbutil"]
   revision = "fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a"
 
-[[projects]]
-  branch = "master"
-  name = "github.com/petar/GoLLRB"
```

Comment:
I'm a little worried that `dep ensure` will just add this back, though.

Reply:
I don't think that is how `dep ensure` works

This is a constraint, telling `dep ensure` that regardless of whether or not anything in the repo actually depends on this, check it out

By removing this, we're no longer forcing dep to check it out, so it deletes it for us

Reply:
https://github.com/golang/dep#removing-dependencies

Reply:
I'm still new to `dep`, but I thought you aren't supposed to edit `Gopkg.lock`, only `Gopkg.toml`.

Reply:
Editing the `toml` was my understanding too

Reply:
Yeah that seems the be what a closer reading of the docs say... Heh